TFT/src/User/API/Language/language_de.h

```diff
@@ -183,6 +184,9 @@
     #define DE_120_SECONDS              "2 Min."
     #define DE_300_SECONDS              "5 Min."
     #define DE_CUSTOM_SECONDS           "Benutz."
-    #define DE_ST7920_FULLSCREEN        "Marlin mode in fullscreen"
+    #define DE_ST7920_FULLSCREEN        "Display mode in fullscreen"
+    #define DE_SOFT_ENDSTOP_OFF         "\u0881 Kein Endstop \u0881"
+    #define DE_MARLIN_MODE              "Display Mode"
+    #define DE_TOUCH_MODE               "Touch Mode"
```

Comment:
the word 'Display' here is ambiguous here.
Marlin Mode / ST7920 Mode/ 12864 Mode are best suited here.

Reply:
yes, you are right. the main purpose is to change the term in every language as you like.
what du you think about "LCD Mode"?

Reply:
“Marlin Mode” and “Touch Mode” is what’s used now to differentiate between the two modes, which works fine.